mlir/test/Conversion/NVVMToLLVM/nvvm-to-llvm.mlir

```diff
@@ -85,93 +85,111 @@ func.func @async_cp_zfill(%dst: !llvm.ptr<3>, %src: !llvm.ptr<1>, %cpSize: i32)
   return
 }
 
+// CHECK-LABEL: @tma_load_4d_all
```

Comment:
can we add just one more of this _all test for 3d as well?

Then, that will cover all cases where im2col is valid + other optional operands.